roles/confluent.common/tasks/debian.yml

```diff
@@ -6,6 +6,14 @@
     gather_subset:
       - '!all'
 
+- name: apt-get update
+  shell: apt-get update
```

Comment:
We should only use shell commands when there isn't an ansible module that can accomplish the same task.  In this instance the apt module should be able to handle this:

https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_module.html